.circleci/integration-tests/master_dag.py

```diff
@@ -62,9 +62,10 @@ def get_report(dag_run_ids: List[str], **context: Any) -> None:
 
         airflow_version = context["ti"].xcom_pull(task_ids="get_airflow_version")
         airflow_executor = context["ti"].xcom_pull(task_ids="get_airflow_executor")
+        astronomer_providers_version = context["ti"].xcom_pull(task_ids="get_astronomer_providers_version")
 
         if IS_RUNTIME_RELEASE:
-            airflow_version_message = f"Results generated for Runtime version {os.environ['ASTRONOMER_RUNTIME_VERSION']} with {airflow_executor}  \n\n"
+            airflow_version_message = f"Results generated for Runtime version {os.environ['ASTRONOMER_RUNTIME_VERSION']} with {airflow_executor} and astronomer-provider version {astronomer_providers_version} \n\n"
         else:
             airflow_version_message = f"The below run is on Airflow version `{airflow_version} with {airflow_executor} executor`\n\n"
```

Comment:
Can we also include the providers version here too, please?

Reply:
@pankajkoti done